sympy/polys/rings.py

```diff
@@ -2531,6 +2560,113 @@ def compose(f, x, a=None):
 
         return poly
 
+    def coeff_split_syms(self, syms):
+        """
+        Split a polynomial into two parts: symbolic and non-symbolic.
```

Comment:
This explanation of "symbolic" and "non-symbolic" makes no sense: all parts have symbols.